ui/src/views/Dashboard.vue

```diff
@@ -1542,11 +1546,13 @@ export default {
         "dashboard.connection_failed"
       );
       const context = this;
+      var streamMessage = "";
       nethserver.exec(
         ["nethserver-ns8-migration/connection/update"],
         validateObj,
-        function(stream) {
-          console.info("ns8-migration-update", stream);
+        function(message) {
+          console.error("ns8-migration-connection-update", message);
```

Comment:
We can avoid writing it to the console.
```suggestion
          console.error("ns8-migration-connection-update", message);
```